bitcoin/psbt.c

```diff
@@ -361,8 +362,8 @@ void psbt_elements_input_set_asset(struct wally_psbt *psbt, size_t in,
 	tal_wally_start();
 
 	if (asset->value > 0)
-		if (wally_psbt_input_set_value(&psbt->inputs[in],
-					       asset->value) != WALLY_OK)
+		if (wally_psbt_set_input_amount(psbt, in,
+						asset->value) != WALLY_OK)
```

Comment:
"Amount" and "value" are the same thing with respect to a transaction input, right?